app/Factory/RecurringInvoiceToInvoiceFactory.php

```diff
@@ -73,8 +74,13 @@ private static function transformItems($recurring_invoice, $client)
         $line_items = $recurring_invoice->line_items;
 
         foreach ($line_items as $key => $item) {
+            $currentDateTime = null;
+            if (isset($recurring_invoice->next_send_date)) {
+                $currentDateTime = Carbon::parse($recurring_invoice->next_send_date);
+            }
+
             if (property_exists($line_items[$key], 'notes')) {
-                $line_items[$key]->notes = Helpers::processReservedKeywords($item->notes, $client);
+                $line_items[$key]->notes = Helpers::processReservedKeywords($item->notes, $client, $currentDateTime);
```

Comment:
You'll also need to construct the currentDateTime with the timezone() method on the carbon instance, ie

```php
->timezone($client->timezone()->name);
```